change/@ni-nimble-components-a8cf147c-e9ea-4478-a258-fba1ad4ae82e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It looks like the button Docs entry is last in its list of stories but all the autogenerated Docs entries are first. Do you know if we can make that consistent? I like having the docs story be selected by default when you click a component name: it feels more like a documentation website and less like a component explorer.

You could try using the `order` configuration described in [Sorting Stories](https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories). It says "stories that don't match an item in the order list will appear after the items in the list" so if we just set something like this configuration we might be good:
```ts
// .storybook/preview.ts

const preview: Preview = {
  parameters: {
    options: {
      storySort: {
        order: ['Getting Started', 'Docs'],
      },
    },
  },
};
```

Reply:
It wasn't quite so simple to fix, but done 👍